### PR TITLE
Add ANCOMBC

### PR DESCRIPTION
**Pull request by d4straub**
We have ANCOM for differential abundance analysis via QIIME2 in the pipeline. However, that is not optimal because rather outdated, allows only pairwise comparisons, and doesnt actually use p-values (valued by many). So I added ANCOM-BC via QIIME2 besides ANCOM, including visualisation of the output. I also made both, ANCOM and ANCOM-BC opt-in choices rather than default, because of the long run time.

Addresses https://github.com/nf-core/ampliseq/issues/731

Still to do:
- [x] test whether combined columns / interactions work with `--ancombc_formula`
- [x] add section to output docu
- [x] add citations to CITATIONS.md
- [x] add section to pipeline summary report
- [x] short paragraph in the usage docu
- [x] update pipeline figure

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/ampliseq/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/ampliseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nf-test test main.nf.test -profile test,docker`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Review by erikrikarddaniel — APPROVED** [review]
Looks good, but I have question. Will approve anyway and leave deciding for you.

File: nextflow_schema.json [hunk 1]
```diff
@@ -681,8 +681,8 @@
                 },
                 "ancombc_args": {
                     "type": "string",
-                    "description": "Extra arguments when performing differential abundance testing with ANCOMBC",
-                    "help_text": "Documentation of arguments can be found in https://docs.qiime2.org/2023.7/plugins/available/composition/ancombc/.",
+                    "description": "Optional extra arguments when performing differential abundance testing with ANCOMBC",
+                    "help_text": "Documentation of arguments can be found in https://docs.qiime2.org/2023.7/plugins/available/composition/ancombc/. For example, the group that serves as reference can be set with `--p-reference-levels`. Please make sure to set all relevant optional parameters that should deviate from the default.",
```
[the remark this answers is not in the document]
Comment (d4straub) [reply nextflow_schema.json:685]: Yes, I assume you are right.
The thing is, that ANCOM-BC will be applied to multiple samplesheet arguments, and I am not sure how that will pan out with reference levels, so I wanted to keep it flexible for now. But probably I should test that in advance...

[the remark this answers is not in the document]
Comment (d4straub) [reply nextflow_schema.json]: I replaced `--ancombc_args` with `--ancombc_formula_reflvl` and added `--ancombc_effect_size` & `--ancombc_significance` for visualisation. Thanks for the feedback!

**Comment by d4straub** [comment]
Thanks!

**Pull request merged**